Makefile

```diff
@@ -13,14 +13,14 @@ install-deps:
 	apt-get install --no-install-recommends -y build-essential clang protobuf-compiler ocl-icd-opencl-dev aria2 cmake
 
 install-lint-tools:
-	which cargo-quickinstall || cargo install cargo-quickinstall
+	cargo install cargo-quickinstall
```

Comment:
this may still fail outside Github workflow, no?

Reply:
Normally `already installed` is not a failure, when $CARGO_HOME/.crates2.json presents, previously we did not cache that file, it's now fixed in the the action

Reply:
Ah, that's the issue and the source of many recent PRs. The `cargo install` behaved differently on my machine. I'm okay with removing `which` if you are confident that there won't be a case where there's a binary but no JSON registry (causing an annoying installation fiasco).

Reply:
@LesnyRumcajs  I've done some testing, it seems quickinstall works fine with the cache action, will just include the removing `which [bin]` change in this PR since it's not yet merged